### PR TITLE
Add Jumperless and Jumperless V5 to supported boards

### PR DESCRIPTION
**Pull request by Architeuthis-Flux**
Hey, I think the hardware for [Jumperless V5](https://www.crowdsupply.com/architeuthis-flux/jumperless-v5) is settled enough that it's time to add it to the supported boards list. 

And @earlephilhower, if you would like a V5 to play with, just let me know.

Thanks for the excellent core and crazy fast RP2350B support!

### CODE REVIEW

File: tools/makeboards.py [hunk 1]
```diff
@@ -501,6 +501,10 @@ def MakeBoardJSON(name, chip, vendor_name, product_name, vid, pid, pwr, boarddef
 MakeBoard("amken_revelop_plus", "rp2040","Amken","Revelop Plus","0x2770",["0x7305"],250,"AMKEN_REVELOP_PLUS",32,0,"boot2_W25Q32JVxQ_4_padded_checksum","","https://www.amken3d.com")
 MakeBoard("amken_revelop_es", "rp2040","Amken","Revelop eS","0x2770",["0x7306"],250,"AMKEN_ES",16,0,"boot2_w25q16jvxq_4_padded_checksum","","https://www.amken3d.com")
 
+# Architeuthis Flux
+MakeBoard("jumperless_v1", "rp2040", "Architeuthis Flux", "Jumperless", "0x1d50", "0xacab", 500, "JUMPERLESS", 16, 0, "boot2_w25q128jvxq_4_padded_checksum","", "https://github.com/Architeuthis-Flux/Jumperless")
+MakeBoard("jumperless_v5", "rp2350", "Architeuthis Flux", "Jumperless V5", "0x1d50", "0xacab", 500, "JUMPERLESS_V5", 16, 0, "boot2_w25q128jvxq_4_padded_checksum","", "https://github.com/Architeuthis-Flux/JumperlessV5")
```
Comment (earlephilhower) [inline tools/makeboards.py:506]: The RP2350 ROM doesn't actually use the `boot2` file.  They seem to have baked in enough intelligence to cover the common cases and any special needs have to be done by the app in flash itself (i.e. in the `variant` init here).

I think given the normal NOR flash you're all good, so could you change this one to `none` like the other 2350 boards?  (Only the 2350...the 2040 still needs and ROM still calls the boot2.S file).

**Review by earlephilhower — CHANGES_REQUESTED** [review]
Minor tweak to `makeboards` and a rerun and you should be good.

Appreciate the offer, but you've already done enough for the project with the 2350 donations.  Having two dozen Pico, Pico2, and ESP32 boards with meters of wire wrap and "Dupont" jumpers on my desk is how I keep my street cred. :laughing:

**Comment by Architeuthis-Flux** [comment]
Haha cool, keep your street cred. The makeboards.py should be fixed, thanks!

**Comment by earlephilhower** [comment]
Change looks good, but somehow you've got a `git` commit changing the RWX bits on `tools/makeboards.py` breaking CI.  I think `git add --chmod=+x tools/makeboards.py` will clear that up...

File: variants/jumperless_v1/pins_arduino.h
Comment (Architeuthis-Flux) [inline variants/jumperless_v1/pins_arduino.h:1]: Ah thanks! I was going crazy trying to figure out what changed there and why that was failing.

**Pull request merged**